test/Microsoft.AspNet.Mvc.TagHelpers.Test/ImageTagHelperTest.cs

```diff
@@ -82,9 +82,6 @@ public void PreservesOrderOfSourceAttributesWhenRun()
             }
         }
 
-        [ConditionalTheory]
```

Comment:
Ohhhh, good catch! l didn't see any errors during the test run so I thought it was running :( I'll fix this tonight.